core/templates/pages/admin-page/admin-page.directive.html

```diff
@@ -1,4 +1,4 @@
-<admin-navbar user-email="$ctrl.userEmail" logout-url="$ctrl.logoutUrl"></admin-navbar>
+<admin-navbar logout-url="$ctrl.logoutUrl"></admin-navbar>
```

Comment:
Could you please remove the email from the `<admin-navbar>` too? Since this is the only place where `<admin-navbar>` is used.

Reply:
Done